specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -194,6 +194,54 @@
           }
         }
       },
+      "put": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "HanaInstances_Create",
+        "x-ms-long-running-operation": true,
+        "summary": "Creates a SAP HANA instance.",
+        "description": "Creates a SAP HANA instance for the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Get properties of a HANA instance": {
+            "$ref": "./examples/HanaInstances_Create.json"
+          }
+        },
+        "parameters": [
```

Comment:
request body?

Reply:
My bad; added to spec, and example updated.

Reply:
@ravbhatnagar Please review